gulpfile.babel.js

```diff
@@ -45,7 +45,8 @@ gulp.task('nodemon', ['copy', 'babel'], () =>
     script: path.join('dist', 'index.js'),
     ext: 'js',
     ignore: ['node_modules/**/*.js', 'dist/**/*.js'],
-    tasks: ['copy', 'babel']
+    tasks: ['copy', 'babel'],
+    legacyWatch: true
```

Comment:
What does this command do?

Reply:
In some networked environments (such as a container running nodemon reading across a mounted drive), you will need to use the legacyWatch: true which enables Chokidar's polling.

https://github.com/remy/nodemon#application-isnt-restarting